java/src/main/java/com/ibm/plugin/rules/detection/jca/JcaDetectionRules.java

```diff
@@ -54,8 +53,6 @@ public static List<IDetectionRule<Tree>> rules() {
                         // secret key
                         JcaSecretKeyFactoryGetInstance.rules().stream(),
                         JcaSecretKeySpec.rules().stream(),
-                        // prng
-                        JcaSecureRandom.rules().stream(),
```

Comment:
Is it voluntary to have removed this without replacing it with `SecureRandomGetInstance` rules?

Reply:
@hugoqnc thanks! For now, yes. This shouldn't be an entry rule. When I refine the java rules, I will but it at the right spot as a depending detection rule.